Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/JumpToBottom.kt

```diff
@@ -60,16 +49,21 @@ fun JumpToBottom(
     modifier: Modifier = Modifier
 ) {
     // Show Jump to Bottom button
-    val transition = transition(
-        definition = definition,
-        toState = if (enabled) Visibility.VISIBLE else Visibility.GONE
-    )
-    if (transition[bottomOffset] > 0.dp) {
+    val transition = updateTransition(if (enabled) Visibility.VISIBLE else Visibility.GONE)
```

Comment:
I think this whole transition could be replaced with an `AnimatedVisibility`?

Reply:
TODO https://github.com/android/compose-samples/issues/370